ipmininet/link.py

```diff
@@ -261,6 +264,29 @@ def __init__(self, node1, node2, intf=IPIntf, *args, **kwargs):
 TCIntf.__bases__ = (IPIntf,)
 
 
+class OrderedAddress(object):
```

Comment:
You may want to look into [functools.total_ordering](https://docs.python.org/3/library/functools.html#functools.total_ordering) to dreduce the boilerplate there.

TLDR:
```python
@functools.total_orderding
class OrderedAddress(object):
    def __init__(self, addr):
        self.addr = addr

    def __lt__(self, other):
        return address_comparator(self.addr, other.addr) < 0

    def __eq__(self, other):
        return address_comparator(self.addr, other.addr) == 0
```